libcaf_net/caf/net/http/request_header.cpp

```diff
@@ -13,6 +13,46 @@ namespace {
 
 constexpr std::string_view eol = "\r\n";
 
+/// Validate the request target part according to RFC9112.
+caf::expected<uri> validate_request_target(http::method method,
+                                           std::string_view request_target) {
+  if (request_target.empty()) {
+    return make_error(sec::invalid_argument,
+                      "Malformed Request-URI: request target empty.");
+  }
+  if (method == http::method::connect) {
+    if (auto res
+        = make_uri(std::string("nil://") + std::string{request_target})) {
+      return std::move(*res);
+    } else {
```

Comment:
```suggestion
    }
```

`else` after return.

Reply:
Local variable `res` is accessed below for logging purposes.